gsuite_reports_rules/gsuite_drive_visibility_change.py

```diff
@@ -1,42 +1,123 @@
 from panther_base_helpers import deep_get
 from panther_base_helpers import gsuite_parameter_lookup as param_lookup
 
+EXCLUDED_DOMAINS = ["runpanther.io"]
+
+VISIBILITY = ["people_with_link", "public_on_the_web", "shared_externally", "unknown"]
```

Comment:
When we use variables like this just to determine membership in a group, we like to use sets (the actual performance considerations are negligible, but it makes it clear to future readers that the intent of the set is to be used in a set like fashion).
```suggestion
VISIBILITY = {"people_with_link", "public_on_the_web", "shared_externally", "unknown"}
```